### DISCUSSION TITLE
0.7.3 changing <Select> focus ring

### DISCUSSION THREAD

**Discussion by DanSizov** [themesberg/flowbite-react#1356] [Q&A] [answered]
![image](https://github.com/themesberg/flowbite-react/assets/111688811/f835a025-ef8e-46b2-a024-2435c1c7c57e)
im trying to change the focus ring color from cyan to indigo-500, custom theme doesnt help, className creates a div-wrapper which parameters also dont work, the version is 0.7.3, i cannot upgrade currently cuz i've tried and i got my componnets css broken completely so i had to downgrade back to 0.7.3, is there any solution for it for this version?

**Comment by SutuSebastian** [comment]
Did u follow the upgrade guide specified in the [release changelog](https://github.com/themesberg/flowbite-react/releases/tag/flowbite-react%400.8.0)?

especially this part:

<img width="1178" alt="Screenshot 2024-04-11 at 12 34 52" src="https://github.com/themesberg/flowbite-react/assets/41998826/3c068d02-d33a-4844-97e8-7052ca71a0b5">

**Comment by SutuSebastian** [comment, marked as answer]
Regarding overriding the border, here's the `theme` config for `select` component overriding the border and focus ring color to `blue`:

```tsx
import { getTheme, Select } from "flowbite-react";

<Select
  theme={{
    field: {
      select: {
        colors: {
          gray: twMerge(
            getTheme().select.field.select.colors.gray,
            "focus:border-blue-500 focus:ring-blue-500 dark:placeholder:text-blue-400 dark:focus:border-blue-500 dark:focus:ring-blue-500",
          ),
        },
      },
    },
  }}
>
  <option>United States</option>
  <option>Canada</option>
  <option>France</option>
  <option>Germany</option>
</Select>
```

Result

<img width="458" alt="Screenshot 2024-04-11 at 13 13 30" src="https://github.com/themesberg/flowbite-react/assets/41998826/e1b1da5f-d2e8-4418-9ae3-1df4dd9c89f7">

Note: the default color is `gray`, but for some reason it has the border and focus ring of `cyan`.

**Reply by DanSizov** [reply]
thank you very much, it works, ill try to upgrade with the guide
